.github/workflows/params-env.yaml

```diff
@@ -6,9 +6,11 @@ on:  # yamllint disable-line rule:truthy
     paths:
       - 'manifests/base/commit.env'
       - 'manifests/base/params.env'
+      - 'manifests/base/commit-latest.env'
+      - 'manifests/base/params-latest.env'
       - 'ci/check-params-env.sh'
-      - 'ci/check-runtime-images.sh'
-      - 'manifests/base/runtime-*-imagestream.yaml'
+      # - 'ci/check-runtime-images.sh'
+      # - 'manifests/base/runtime-*-imagestream.yaml'
```

Comment:
hm, do we want this to be commented out? I think that it's still good to have it here or am I missing something? :thinking:

Reply:
as we moved the image quotient to params.env , realized running this workflow on change of runtime-imagestream.yaml should have no impact.

Reply:
ah, right, we probably don't care about anything else there... I see, you are right, thanks!